docusaurus/docs-internals/editor-controls/data-controls/fileUpload.md

```diff
@@ -176,3 +238,90 @@ In this case, the `fileUpload` control allows uploading only `.png,` `.jpg,` `.j
   }
 }
 ```
+
+#### CSS examples
+
+Show or hide the `.brz-download` element with CSS using a `multiSelect` control value.
+If `value.id` exist, then the color of `.brz-download` will be `green`;
+
+```js
+{
+  id: "file",
+  type: "fileUpload",
+  style: ({ value }) => {
+    if (value.id) {
```

Comment:
you has no `value.id how your value look:
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/a9be5f7a-68e3-479c-a993-cf3be4210c55)